diff --git a/node_modules/react-native/Libraries/Utilities/Appearance.js b/node_modules/react-native/Libraries/Utilities/Appearance.js
index 3cd23a6..d96047b 100644
--- a/node_modules/react-native/Libraries/Utilities/Appearance.js
+++ b/node_modules/react-native/Libraries/Utilities/Appearance.js
@@ -105,7 +105,9 @@ export function setColorScheme(colorScheme: ?ColorSchemeName): void {
   const {NativeAppearance} = state;
   if (NativeAppearance != null) {
     NativeAppearance.setColorScheme(colorScheme ?? 'unspecified');
-    state.appearance = {colorScheme};
+    state.appearance = {
+      colorScheme: toColorScheme(NativeAppearance.getColorScheme()),
+    };
   }
 }
 
